text/0033-view-hierarchy.md

```diff
@@ -10,12 +10,69 @@ Allow SDKs to send a textual representation of the application's UI view hierarc
 
 # Motivation
 
-Being able to see what the user was looking at is a good way to found out problems in the application. 
-This RFC proposes a way to send information from the UI state of an application, known as "view hierarchy".
+Being able to see what the user was looking at is a good way to find problems in an application. This RFC proposes a way to send information from the UI state of an application, known as "view hierarchy". Compared to screenshots the view hierarchy provides complementary information about the UI, e.g. relationships between UI widgets or additional attributes not visible on the screen.
 
-# Proposal 
+Example View Hierarchies from PoC implementations:
+* [Android](https://github.com/getsentry/sentry-java/pull/1998#issuecomment-1104806065)
+* [iOS](https://github.com/getsentry/sentry-cocoa/pull/2044#issue-1329719543)
+* [Flutter](https://github.com/ueman/sentry-dart-tools/blob/b850db587f0e099bed253d13055c88c03d536875/sentry_flutter_plus/lib/src/integrations/tree_walker_integration.dart)
+* [Unity](https://github.com/getsentry/team-mobile/issues/64#issuecomment-1290868653)
 
-The proposal is to create a new [envelope item](https://develop.sentry.dev/sdk/envelopes/) in the form of a json representation of the UI elements. 
+# Proposal
+
+Capture the view hierarchy in our mobile SDKs and convert it to a common JSON representation. Add it as an [attachment](https://develop.sentry.dev/sdk/envelopes/#attachment) to the envelope. The `attachment_type` is set to `"event.viewhierarchy"` and the `content_type` is set to the `"application/json"` mime-type.
+
+The attachment payload is a JSON structure containing the view hierarchy, here's an example:
+```json
+{
+    "rendering_system": "compose", // or android_view_system, apple_uikit, apple_swiftui, unity, flutter, ...
+    "screens": [
```

Comment:
@markushi I think windows is more correct, screens is more related to displays imo.

```suggestion
    "windows": [
```